client/src/components/BizCard_Map/BizCard_Map.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState } from 'react';
+import { GoogleMap, Marker, InfoWindow } from '@react-google-maps/api';
+import flag from '../../images/hopper_flag.png'; 
+
+
+function BizCard_Map({props}){
```

Comment:
once you rename `props` to `biz`, all the `props` names in this component should be switched to `biz`

Reply:
I changed from props to biz.